content/community/conferences.md

```diff
@@ -12,6 +12,11 @@ Do you know of a local React.js conference? Add it here! (Please keep the list c
 
 ## Upcoming Conferences {#upcoming-conferences}
 
+### Reactconf IRAN 2019 {#ReactConfIran2019}
+AUGUST 29, 2019. TEHRAN, IRAN.
```

Comment:
Do you mind writing it without caps lock? Same as other entries.

Reply:
thanks, it's fixed.